torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6917,6 +6934,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            # the gradient check for complex fails.
            backward_dtypes=floating_types_and(torch.float16, torch.bfloat16),
            ),
+    OpInfo('attn',
+           dtypes=floating_types_and(torch.bfloat16),
+           sample_inputs_func=sample_inputs_attn,
+           skips=(
+               SkipInfo('TestCommon', 'test_out'),
```

Comment:
You have a `supports_out` setting here that you can set to False instead of skipping the test manually